tests/executors/indexers/vector/test_annoy.py

```diff
@@ -56,6 +57,35 @@ def test_simple_annoy(self):
         _index.build(4)
         idx1, _ = _index.get_nns_by_vector(np.random.random((5,)), 3, include_distances=True)
 
+    def test_simple_ngt(self):
```

Comment:
Could you alse add a test where your NGTIndexer is used?

Reply:
@JoanFM Sorry for the late reply , I created the NGTIndexer test